docs/content/intro/known-limitations.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Known Limitations"
+date: 2018-11-01T13:58:58-07:00
+weight: 4
+LastModifierDisplayName: "Marwan"
+LastModifierEmail: "marwan.sameer@gmail.com"
+
+---
+
+Go Modules is still behind an experimental flag and therefore it's important to know how Athens can help you *now* until all issues in the upstream of Go modules are resolved. 
+
+#### What Athens can do well _now_ 
+
+- Athens will work well within your CI/CD system that installs your modules without needing a vendor folder.
```

Comment:
for me future tense evokes that it will but it's not there yet, while the title says now

Reply:
got it, will change to `Athens works well`